builder/builder.go

```diff
@@ -720,3 +733,50 @@ func checkRPM(path string) error {
 	}
 	return nil
 }
+
+// stopWatch keeps track of a sequence of durations. Use Start and Stop to mark the sections, then
+// write the final result with WriteSummary.
+type stopWatch struct {
```

Comment:
These things seem useful enough to put in their own file or in helpers.go to me.

Reply:
Will move to its own file.